desktop-environment.el

```diff
@@ -411,12 +421,21 @@ the screen."
 
 (declare-function exwm-input-set-key "ext:exwm-input")
 
-(defun desktop-environment-exwm-set-global-keybindings ()
+(defun desktop-environment-exwm-set-global-keybindings (enable)
   "When using EXWM, add `desktop-environment-mode-map' to global keys."
   (when (featurep 'exwm-input)
-    (map-keymap (lambda (event definition)
-                  (exwm-input-set-key (vector event) definition))
-                desktop-environment-mode-map)))
+    (cond ((eq desktop-environment-update-exwm-global-keys :global)
```

Comment:
you could use `pcase` or `cl-case` here if you want.

Reply:
That would add dependencies, right?

Reply:
Not sure what you mean exactly. Both are provided with Emacs so no external dependency is necessary. Both are autoloaded so no `require` form should be added. Additionally, `cl-case` doesn't even require `cl-lib` to be available at runtime.

Reply:
Ah ok.  Will use `cl-case`, then.